packages/augur-simplified/src/assets/styles/_colors.less

```diff
@@ -23,6 +23,7 @@
   --color-primary-text-30: fade(#D7DDE0, 30%);
   --color-primary-text-40: fade(#D7DDE0, 40%);
   --color-primary-text-50: fade(#D7DDE0, 50%);
+  --color-primary-text-80: fade(#D7DDE0, 70%);
```

Comment:
Is this even used? if so, rename it to --color-primary-text-70 (already have an 80) if it's not in use then just remove it.